lib/config/options/index.ts

```diff
@@ -343,6 +343,12 @@ const options: RenovateOptions[] = [
     globalOnly: true,
     default: 'renovate_',
   },
+  {
+    name: 'dockerCliOptions',
+    description: 'Docker run command options for use when `binarySource=docker`.',
```

Comment:
This description needs to be improved as well. It feels a bit too short for me.

Reply:
not sure what to add🤔